includes/functions.php

```diff
@@ -1537,35 +1538,7 @@ function is_activity( $data ) {
 	 */
 	$types = apply_filters(
 		'activitypub_activity_types',
-		array(
-			'Accept',
-			'Add',
-			'Announce',
-			'Arrive',
-			'Block',
-			'Create',
-			'Delete',
-			'Dislike',
-			'Follow',
-			'Flag',
-			'Ignore',
-			'Invite',
-			'Join',
-			'Leave',
-			'Like',
-			'Listen',
-			'Move',
-			'Offer',
-			'Read',
-			'Reject',
-			'Remove',
-			'TentativeAccept',
-			'TentativeReject',
-			'Travel',
-			'Undo',
-			'Update',
-			'View',
-		)
+		Activity::DEFAULT_TYPES
```

Comment:
With this replacement we should be able to bring the whole `apply_filters()` call into one line.